dpl-chef_supermarket.gemspec

```diff
@@ -9,9 +9,9 @@ if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.3.0")
     deps << ['net-telnet', '~> 0.1.0'] << ['chef', '~> 12.0'] << ['public_suffix', '< 3.1.0']
     gemspec_for 'chef_supermarket', deps
 elsif Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.4.0")
-  gemspec_for 'chef_supermarket', (deps << ['chef', '~> 13.0'])
+  gemspec_for 'chef_supermarket', (deps << ['chef', '~> 13.0'] << ['openssl'])
```

Comment:
wont all of these resolve based on the system that builds the spec rather than runtime? If your talking about conditionally wanting to install a gem based on the version of ruby you probably need something ugly like this: https://github.com/sensu-plugins/sensu-plugins-supervisor/pull/22

Reply:
You are correct to point out that there is a mismatch between the dependencies we define here, and the dependencies defined by the published gem.